### PR TITLE
Update pricing details on the support tiers

### PR DESCRIPTION
**Pull request by iconara**
The developer support was recently changed to be the greater of $29 or 3% of your bill.

Also change from "at least 10%" to "up to 10%" in the next paragraph. No support plan adds more than 10% to your bill.

### CODE REVIEW

**Comment by richadams** [comment]
The pricing detail page might be a good link to include as it explains it with some examples: https://aws.amazon.com/premiumsupport/pricing/

Is it worth mentioning that older accounts can still get the original pricing if they already had it set up?

File: README.md [hunk 1]
```diff
@@ -371,8 +371,8 @@ It’s important to know the maturity of each AWS product. Here is a mostly comp
 
 -	**Forums:** For many problems, it’s worth searching or asking for help in the [discussion forums](https://forums.aws.amazon.com/index.jspa) to see if it’s a known issue.
 -	**Premium support:** AWS offers several levels of [premium support](https://aws.amazon.com/premiumsupport/).
-	-	Any small company should probably pay for the cheap “Developer” support as it’s a flat $49/month and it lets you file support tickets with 12 to 24 hour turnaround time.
-	-	The higher-level support services are quite expensive — and increase your bill by at least 10%. Many large and effective companies never pay for this level of support. They are usually more helpful for midsize or larger companies needing rapid turnaround on deeper or more perplexing problems.
+	-	The first tier, called "Developer support" lets you file support tickets with 12 to 24 hour turnaround time, it starts at $29 but once your monthly spend reaches around $1000 it changes to a 3% surcharge on your bill.
```
Comment (jlevy) [inline README.md]: How about mentioning legacy price as a second sentence?

Comment (jlevy) [inline README.md]: Well, in general, if you can't find a fact elsewhere easily, there's some value in having it here.

That said, if legacy is also going away in 6 months, I'm fine with omitting it.

**Comment by jlevy** [comment]
👍 Thanks!

**Pull request merged**